setup.sh

```diff
@@ -162,6 +159,26 @@ function confirm_complete_removal()
   fi
 }
 
+# This function moves the folders from the old directory structure to
+# folders of the new one.
+function legacy_folders()
+{
+  local prefix="$HOME/.local"
+
+  if [[ -d "$prefix/share/doc" && -f "$prefix/share/man/kw.rst" &&
+    -d "$prefix/share/sound/kw" && -d "$prefix/etc/kw" &&
+    -d "$HOME/.kw" ]]; then
+
+    mv "$HOME/.kw/"* "$datadir"
+    rm -rf "$prefix/share/doc"
+    rm -rf "$prefix/share/man/kw.rst"
+    rm -rf "$prefix/share/sound/kw"
+    mv "$prefix/etc/kw" "$etcdir"
```

Comment:
In the `mv` cases, let's add a message that tells the user that we changed those file locations from X to Y.